R/deps.R

```diff
@@ -109,7 +109,7 @@ dev_package_deps <- function(pkg = ".", dependencies = NA,
   deps <- unlist(lapply(parsed, `[[`, "name"), use.names = FALSE)
 
   if (is_bioconductor(pkg)) {
-    check_suggested("BiocInstaller")
+    check_bioconductor()
```

Comment:
Is it necessary to stop here on failure? There appear to be CRAN packages for which `is_bioconductor()` is true